datvuthanh_hybridnets.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: HybridNets
+summary: HybridNets - End2End Perception Network
+image: hybridnets.jpg
+author: Dat Vu Thanh
+tags: [vision]
+github-link: https://github.com/datvuthanh/HybridNets
+github-id: datvuthanh/HybridNets
+featured_image_1: no-image
+featured_image_2: no-image
+accelerator: cuda-optional
+demo-model-link: https://colab.research.google.com/drive/1Uc1ZPoPeh-lAhPQ1CloiVUsOIRAVOGWA
+---
+## Before You Start
+
+Start from a **Python>=3.7** environment with **PyTorch>=1.10** installed. To install PyTorch see [https://pytorch.org/get-started/locally/](https://pytorch.org/get-started/locally/). To install HybridNets dependencies:
+```bash
+pip install -qr https://raw.githubusercontent.com/datvuthanh/HybridNets/main/requirements.txt  # install dependencies
+```
+
+## Model Description
+ 
+<img src="https://github.com/datvuthanh/HybridNets/raw/main/images/hybridnets.jpg">  
```

Comment:
Might be a display issue from netlify or something on my side but this image appears to be quite big on https://pytorch.org/hub/, perhaps could you reshape it with `width="400"`  or something like it?

Reply:
Oh sorry for not noticing that, it's on my end too. I'll try `width="800"`.